chainstate/src/rpc.rs

```diff
@@ -77,6 +81,11 @@ impl ChainstateRpcServer for super::ChainstateHandle {
         handle_error(self.call(move |this| this.get_block_id_from_height(&height)).await)
     }
 
+    async fn block(&self, id: Id<Block>) -> rpc::Result<Option<String>> {
```

Comment:
Please call it get_block

Reply:
I had my doubts about the naming, but for some reason other RPC methods (`best_block_id`, `block_id_at_height`, etc.) don't have the `get_` prefix. Should I rename them too?

Reply:
The other ones seem OK. "Block" alone seems to be not meaningful enough in rpc.